src/vehicle.cpp

```diff
@@ -1394,12 +1394,7 @@ bool vehicle::fold_up() {
     }
 
     // create a folding [non]bicycle item
-    item bicycle;
-    if (can_be_folded) {
-        bicycle.make( "generic_folded_vehicle" );
-    } else {
-        bicycle.make( "folding_bicycle" );
-    }
+    item bicycle( can_be_folded ? "generic_folded_vehicle" : "folding_bicycle", calendar::turn );
```

Comment:
* Start vehicle construction from build menu
* Add folding frame to it
* Remove the initial non-folding frame
* Examine the vehicle made of one folding frame
* Fold it
* Crash

Compiled with g++, release build. May be one of "those bugs" where pointer wonkery means it only crashes in some cases.